posts/2021-01-06-ocp-log-forwarding.adoc

```diff
@@ -386,18 +396,20 @@ oc delete pod --selector logging-infra=fluentd
 
 Note: After you set the `managementState` to "Unmanaged" in the _ClusterLogging_ instance, any further changes to the _ClusterLogging_ or _ClusterLogForwarder_ instances are not automatically detected. If you need to modify the configuration for either instance, you must change the managementState field back to "Managed". However, after the `managementState` is changed to "Managed", the _ClusterLogging_ instance reverts the Fluentd ConfigMap back to its original settings. You must complete the previous steps again to re-enable Fluentd to parse JSON.
 
-== Setting up the Splunk dashboard and viewing logs
+== Viewing logs in Splunk
 
 
-. Go to Search & Reporting. Search for `index="openshift"` to view logs from OpenShift Container Platform.
+To view the logs forwarded from the OpenShift platform go to _Search & Reporting_. Search for `index="openshift"` to view logs from OpenShift Container Platform.
 
-[start=2]
+If you have configured Open Liberty with JSON logging and OpenShift's Fluentd collectors to parse the JSON logs you will be able to use Open Liberty's Splunk dashboards.
```

Comment:
If you have configured Open Liberty with JSON logging and OpenShift's Fluentd collectors to parse the JSON logs you will be able to use Open Liberty's Splunk dashboards.

-->

If you configure Open Liberty with JSON logging and the OpenShift Fluentd collectors to parse the JSON logs, you can use the Open Liberty Splunk dashboards.